src/DynamicRupture/Initializers/RateAndStateInitializer.cpp

```diff
@@ -189,16 +71,14 @@ std::pair<real, real>
                                                             real initialSlipRate) {
   real absoluteTraction = std::sqrt(std::pow(tractionXY, 2) + std::pow(tractionXZ, 2));
   real tmp = std::abs(absoluteTraction / (rs_a * pressure));
-  real stateVariable = rs_a / rs_sr0 *
+  real stateVariable = rs_sl0 / rs_sr0 *
                        std::exp((rs_a * std::log(std::exp(tmp) - std::exp(-tmp)) - rs_f0 -
                                  rs_a * std::log(initialSlipRate / rs_sr0)) /
-                                drParameters.rs_b);
+                                rs_b);
   real tmp2 = initialSlipRate * 0.5 / rs_sr0 *
-              std::exp((drParameters.rs_f0 +
-                        drParameters.rs_b * std::log(drParameters.rs_f0 * stateVariable / rs_sl0)) /
-                       rs_a);
+              std::exp((rs_f0 + rs_b * std::log(rs_sr0 * stateVariable / rs_sl0)) / rs_a);
```

Comment:
std::log(rs_f0

Reply:
Again, I think the error was in the initial state, c.f. https://github.com/SeisSol/SeisSol/blob/bc4b4625fb1be72ccd370a30aa6e3c3351e006ed/src/Physics/ini_model_DR.f90#L678

Reply:
ok, sorry, I just thought you may have changed the variable name by mistake.
Yes that's correct.

Reply:
Always good to double check ;-)